stdlib/public/core/CommandLine.swift

```diff
@@ -54,11 +54,44 @@ public enum CommandLine {
     return _unsafeArgv
   }
 
-  /// Access to the Swift command line arguments.
-  // Use lazy initialization of static properties to 
+  // Use lazy initialization of static properties to
   // safely initialize the swift arguments.
-  public static var arguments: [String]
+  @usableFromInline
+  internal static var _arguments: [String]
     = (0..<Int(argc)).map { String(cString: _unsafeArgv[$0]!) }
+ 
+  /// An array that provides access to this program's command line arguments.
+  ///
+  /// Use `CommandLine.arguments` to access the command line arguments used
+  /// when executing the current program. The name of the executed program is
+  /// the first argument.
+  ///
+  /// The following example shows a command line executable that squares the
+  /// integer given as an argument.
+  ///
+  ///     if CommandLine.arguments.count == 2,
+  ///         let number = Int(CommandLine.arguments[1])
+  ///     {
+  ///         print("\(number) x \(number) is \(number * number)")
+  ///     } else {
+  ///         print("""
+  ///           Error: Please provide a number to square.
+  ///           Usage: command <number>
+  ///           """)
+  ///     }
+  ///
+  /// Running the program results in the following output:
+  ///
+  ///     $ command 5
+  ///     5 x 5 is 25
+  ///     $ command ZZZ
+  ///     Error: Please provide a number to square.
+  ///     Usage: command <number>
+  public static var arguments: [String] {
```

Comment:
I think this needs to be marked `@_alwaysEmitIntoClient`

Reply:
What are the ABI stability implications of doing that? Existing code will have references to `arguments` that were _not_ emitted into their binaries; does that work with `@_alwaysEmitIntoClient`? And presumably we're OK with making `_arguments` into ABI if we're doing that?

Reply:
I misunderstood why the ABI test failed! It's the accessor symbols for `_arguments` that were added.

The typical pattern for this is to leave the ABI declaration exactly as-is and change `public` to `@usableFromInline internal`, and make the new public API `@_alwaysEmitIntoClient`. But I think this change here is conceptually fine because changing a stored property to a computed one is ABI compatible, and the new `_arguments` variable is only used on newer runtimes while old runtimes will use the stored variable accessors directly.